<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgScreenPos</class>
 <widget class="QDialog" name="DlgScreenPos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chXZ">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set width size to full screen</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chYL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>absolute Y position on screen</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chZB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set bottom z order of window</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chYZ">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set height  size to full screen</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chXL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>absolute X position on screen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>KEYPAD POSITION ON SCREEN</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chZT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set top z order of window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgScreenPos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgScreenPos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
